### PR TITLE
KevS-EMLyon-subdomains-TRANSLATIONS

### PR DESCRIPTION
**Pull request by Kazarus4**
Translations doing

### CODE REVIEW

**Comment by Kazarus4** [comment]
All translations are done.

Need proof by : 
- @tcpdumpfbacke for DE and EN
- @Y0Coss for latin and PL

File: pages/web_cloud/domains/domain_create_subdomains/guide.es-es.md [hunk 1]
```diff
@@ -94,7 +94,7 @@ Existen dos situaciones posibles:
 >
 > 1: Para identificar la zona DNS activa de un dominio registrado en OVHcloud, consulte nuestra guía "[Cambiar los servidores DNS de un dominio en OVHcloud](/pages/web_cloud/domains/dns_server_general_information)".
 >
-> 2: Si su dominio no está registrado con OVHcloud, contacte con el actual agente registrador* para conocer el lugar en el que está alojada su zona DNS activa.
+> 2: Si su dominio no está registrado con OVHcloud, contacte con el actual agente registrador* para conocer el lugar en el que está alojada su zona DNS activa. Si desea realizar esta acción, puede consultar nuestra guía "[Transferir un dominio a OVHcloud](/pages/web_cloud/domains/transfer_incoming_generic_domain)".
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/domains/domain_create_subdomains/guide.es-es.md]:
```suggestion
> 2: Si su dominio no está registrado con OVHcloud, contacte con el actual agente registrador* para conocer el lugar en el que está alojada su zona DNS activa. Puede consultar nuestra guía "[Transferir un dominio a OVHcloud](/pages/web_cloud/domains/transfer_incoming_generic_domain)" si desea realizar esta acción.
```

File: pages/web_cloud/domains/domain_create_subdomains/guide.es-us.md [hunk 2]
```diff
@@ -94,7 +94,7 @@ Existen dos situaciones posibles:
 >
 > 1: Para identificar la zona DNS activa de un dominio registrado en OVHcloud, consulte nuestra guía "[Cambiar los servidores DNS de un dominio en OVHcloud](/pages/web_cloud/domains/dns_server_general_information)".
 >
-> 2: Si su dominio no está registrado con OVHcloud, contacte con el actual agente registrador* para conocer el lugar en el que está alojada su zona DNS activa.
+> 2: Si su dominio no está registrado con OVHcloud, contacte con el actual agente registrador* para conocer el lugar en el que está alojada su zona DNS activa. Si desea realizar esta acción, puede consultar nuestra guía "[Transferir un dominio a OVHcloud](/pages/web_cloud/domains/transfer_incoming_generic_domain)".
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/domains/domain_create_subdomains/guide.es-us.md]:
```suggestion
> 2: Si su dominio no está registrado con OVHcloud, contacte con el actual agente registrador* para conocer el lugar en el que está alojada su zona DNS activa. Puede consultar nuestra guía "[Transferir un dominio a OVHcloud](/pages/web_cloud/domains/transfer_incoming_generic_domain)" si desea realizar esta acción.
```

File: pages/web_cloud/domains/domain_create_subdomains/guide.it-it.md [hunk 3]
```diff
@@ -94,7 +94,7 @@ Sono possibili due scenari:
 >
 > 1: Per identificare la zona DNS attiva di un dominio registrato in OVHcloud, consulta la nostra guida "[Modificare i server DNS di un dominio OVHcloud](/pages/web_cloud/domains/dns_server_general_information)".
 >
-> 2: se il dominio non è registrato in OVHcloud, contatta il *Registrar* attuale per conoscere la zona DNS attiva.
+> 2: se il dominio non è registrato in OVHcloud, contatta il *Registrar* attuale per conoscere la zona DNS attiva. Per effettuare questa operazione, utilizza la nostra guida "[Trasferire un dominio in OVHcloud](/pages/web_cloud/domains/transfer_incoming_generic_domain)".
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/domains/domain_create_subdomains/guide.it-it.md]: That translations seems misleading. 
```suggestion
> 2: se il dominio non è registrato in OVHcloud, contatta il *Registrar* attuale per conoscere la zona DNS attiva. È possibile trasferire il dominio in OVHcloud seguendo la nostra [guida dedicata](/pages/web_cloud/domains/transfer_incoming_generic_domain).
```

File: pages/web_cloud/domains/domain_create_subdomains/guide.pl-pl.md [hunk 4]
```diff
@@ -94,7 +94,7 @@ Możliwe są dwa przypadki:
 >
 > 1 : Aby ustalić, gdzie znajduje się aktywna strefa DNS domeny zarejestrowanej w OVHcloud, skorzystaj z naszego przewodnika "[Modyfikacja serwerów DNS domeny OVHcloud](/pages/web_cloud/domains/dns_server_general_information)".
 >
-> 2: Jeśli Twoja domena nie jest zarejestrowana w OVHcloud, skontaktuj się z *aktualnym operatorem* Twojej domeny, aby dowiedzieć się, gdzie hostowana jest aktywna strefa DNS.
+> 2: Jeśli Twoja domena nie jest zarejestrowana w OVHcloud, skontaktuj się z *aktualnym operatorem* Twojej domeny, aby dowiedzieć się, gdzie hostowana jest aktywna strefa DNS. Pamiętaj, że możesz skorzystać z naszego przewodnika "[Transfer_domeny do OVHcloud](/pages/web_cloud/domains/transfer_incoming_generic_domain)", jeśli chcesz wykonać tę czynność.
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/domains/domain_create_subdomains/guide.pl-pl.md]:
```suggestion
> 2: Jeśli Twoja domena nie jest zarejestrowana w OVHcloud, skontaktuj się z *aktualnym operatorem* Twojej domeny, aby dowiedzieć się, gdzie hostowana jest aktywna strefa DNS. Aby wykonać transfer domeny do OVHcloud, zapoznaj się z naszym [przewodnikiem](/pages/web_cloud/domains/transfer_incoming_generic_domain).
```

**Review by OVH-Team-Guides — APPROVED** [review]

**Review by bs-ovh — APPROVED** [review]

**Pull request merged**